src/hrflow_connectors/connectors/leboncoin/connector.py

```diff
@@ -25,36 +26,44 @@
 
 
 def get_job_location(location: t.Dict) -> t.Dict:
-    if location.get("text"):
-        location_text = location["text"]
+    # get location from location dictionary
+    zip_code = None
+    city = None
+    country = None
+    # best case is when location["fields"] is not empty
+    # due to a bug in the API fields is a list instead of a dict
+    if (location["fields"]) is t.Dict:
```

Comment:
This do work, but It's better to use built-in function isinstance(object, type) cf https://docs.python.org/3/library/functions.html#isinstance

if isinstance(location["fields"], dict):